bundle/jsky.app.ot/src/main/scala/jsky/app/ot/itc/ConditionsPanel.scala

```diff
@@ -63,7 +63,8 @@ final class ConditionsPanel(owner: EdIteratorFolder) extends GridBagPanel {
     cc.selection.item,
     wv.selection.item,
     sb.selection.item,
-    am.selection.item)
+    am.selection.item, 0.0, 0.0
```

Comment:
Does this mean in the OT we won't see the option to set these fields?

Reply:
We would need to modify the OT UI which we probably don't want to do?